+euphonic/private/euphonic_on.m

```diff
@@ -0,0 +1,45 @@
+function euphonic_on()
+
+global horace_euphonic_interface_is_initialised;
+if isempty(horace_euphonic_interface_is_initialised) || ~horace_euphonic_interface_is_initialised
+
+% Set number of openblas threads to 1 so it doesn't interfere with openmp
+setenv('OPENBLAS_NUM_THREADS', '1');
+
+% Sets library opening flag on Linux to load C modules immediately and to
+% force Matlab to use the math libraries of this module instead of its own.
+if ~ispc
+    py.sys.setdlopenflags(int32(10));
+end
+
+% Check if Euphonic is installed and the correct version
+req_mods = required_modules;
+verify_python_modules(req_mods{:});
```

Comment:
If the correct modules aren't installed, this will print something like:

`Python Error: DistributionNotFound: The 'euphonic' distribution was not found and is required by the application`

which isn't very helpful for the user. Maybe catch these errors and rethrow with some additional helpful text? (e.g. what Python executable is being used, how to install the correct modules)